LearningMissionSimulation/PlaygroundGarush.cs

```diff
@@ -124,7 +161,11 @@ public void CreateModules(int moduleCount)
                 moduleIdList.Add(module.Id);
                 Console.WriteLine("\ncreate module {0}\n", module);
                 i++;
+                ReportItem(itemName: module.ToString(), actionName: action, itemIndex: i);
```

Comment:
Combine both lines:
                i++;
                ReportItem(itemName: module.ToString(), actionName: action, itemIndex: i);
in one:
                ReportItem(itemName: module.ToString(), actionName: action, itemIndex: ++i);